infra/recipes/docker-compose/common/oauth/fetch_token.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Check if username is provided
+if [ -z "$1" ]; then
+    echo "Error: Username not provided."
+    echo "Usage: $0 <username>"
+    exit 1
+fi
+
+KEYCLOAK_URL="http://localhost:8085"
+REALM="openhouse"
+CLIENT_ID="openhouse-services"
+USERNAME="$1"
+PASSWORD="$1"
```

Comment:
Nit: Should this be different? Right now username and password seems to be the same but should that be handled separately?

Reply:
I kept them same, given this is for local docker environment. Do you see an issue with that from keycloak or oauth token standpoint?

Reply:
It does not seem like an issue for keycloak.